### ISSUE TITLE
Provide option to specify category title manually, rather than automatically generated

### ISSUE THREAD

**Issue by caseyjhol** [gilbitron/Raneto#65]
This would be handy for custom capitalization and special characters, or just for having shorter URLs (e.g. **/faqs/...** instead of **/frequently-asked-questions/...** A simple method would probably be to include a `title` file in the category folder with the custom title (similar to the [way categories are sorted](http://docs.raneto.com/usage/sorting)).

**Comment by ryanlelek** [comment]
Related comment from #18 
> Should we do a central YAML file per category and put all category, article, and sort metadata in there?

Would that work for your purposes @caseyjhol?

**Comment by caseyjhol** [comment]
Yes, that work, and would be much easier than separating it into multiple files.

**Comment by namannik** [comment]
I second this.  For our support documentation, our company name, hearTV, is being rendered as "Hear TV".

**Comment by forsureitsme** [comment]
I third this, because using Raneto with pt_BR titles, it just can't find categories named with characters like `ç`, `ã`, `á` and so on...

**Comment by ryanlelek** [comment]
I agree. My bandwidth is low right now, so it's up for grabs.
Marking it as a priority tag

**Comment by IllusionaryOne** [comment]
I made a kludge in which a "dirmeta" file is created underneath the newly created category directory.  That file contains the raw input from the category creation before it is sanitized for the file system.  I am happy to submit the change as a pull request if this sounds like a decent way to accomplish this.

**Comment by ryanlelek** [comment]
Yes, that would be a great start to a solution @IllusionaryOne.
Happy to review and discuss the PR when you submit

**Comment by forsureitsme** [comment]
Yes, do it @IllusionaryOne. We can help you turning the meta into YAML for the issue #18

**Comment by Evgeny-A** [comment]
This issue seems to be fixed. Why isn't it closed?
